src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -447,6 +469,65 @@ describe('non-empty successful affected clusters table', () => {
       .eq(0)
       .should('have.text', 'dd2ef343-9131-46f5-8962-290fdfdf2199');
   });
+
+  // TODO fix test: double sorting? keep ordering in single sorting?
+  Object.entries({
+    name: 'Name',
+    last_checked_at: 'Last seen',
+  }).forEach(([category, label]) => {
+    ['ascending', 'descending'].forEach((order) => {
+      it(`sort ${order} by ${label}`, () => {
+        const col = `td[data-label="${label}"]`;
+        const header = `th[data-label="${label}"]`;
+
+        cy.get(col).should(
+          'have.length',
+          Math.min(DEFAULT_ROW_COUNT, data['enabled'].length)
+        );
+        cy.get(header).find('button').click();
+        // FIXME right way to do the second click?
```

Comment:
https://docs.cypress.io/api/commands/dblclick ?

Reply:
Nice, thank you! I missed it